src/milia/api/organization.cljc

```diff
@@ -107,6 +108,26 @@
                  :suppress-4xx-exceptions? true
                  :as-map? true))))
 
+(defn change-org-member-role
+  "Change the role of an organization member"
+  [member org-name event-chan add-callback?]
+  (let [data {:username (:username member)
+              :role (:role member)}]
+    (apply parse-http (->
+                        [:put
+                         (make-url "orgs" org-name "members")
+                         :http-options
+                         #?(:clj   {:form-params data
+                                    :content-type :json})
+                         #?(:cljs {:json-params data})
+                          :as-map? true]
+                        (concat (when add-callback?
+                                  [:callback
+                                   #?(:cljs
```

Comment:
I think you want to put `:cljs` at a higher level, so you don't have any `:callback` key for `:clj`, otherwise don't need to both with the apply can just have `nil` other the `:clj` branch or something

Reply:
Am not sure i got your point here :) but I have used the `add-callback?` check which can be `false`  for `:clj` thus not adding any `:callback` key. Does this make sense?

Reply:
Not really, since if we're the compiler we'll know whether it's for `clj` or `cljs` and shouldn't have to involve any additional vars. Probably best to simply pass a `:clj` nil and have it ignored by the Clojure version of `parse-http`, i.e.

```clojure
(defn change-org-member-role
  "Change the role of an organization member"
  [member org-name event-chan]
  (let [data {:username (:username member)
              :role (:role member)}]
    (parse-http :put
                (make-url "orgs" org-name "members")
                :http-options
                #?(:clj  {:form-params data
                          :content-type :json})
                   :cljs {:json-params data})
                          :as-map? true]
                :callback
                #?(:clj nil
                   :cljs #(put! event-chan
                                {:updated-member member}))]))))))

Reply:
Okay cool, this can work too thanks.